README.md

```diff
@@ -10,7 +10,7 @@
 
 ## Introduction
 
-This repo contains Terraform templates to help deployment of OpenShift Container Platform (OCP) 4.x on [IBM® Power Systems™ Virtual Server on IBM Cloud](https://www.ibm.com/cloud/power-virtual-server).
+This repo contains Terraform templates to help deployment of OpenShift Container Platform (OCP) 4.6.x releases on [IBM® Power Systems™ Virtual Server on IBM Cloud](https://www.ibm.com/cloud/power-virtual-server).
```

Comment:
The repository is actually for OCP 4 and the master "branch" is 4.6+. We can mention in here that to install OCP 4.5 and lower versions use release-4.5 branch and OCP 4.6 and higher version use the master branch.

Reply:
Now I see your PR for the master branch. Ignore the above comment.